atecc608a_utils.c

```diff
@@ -61,3 +61,20 @@ psa_status_t atecc608a_check_config_locked()
     }
     return status;
 }
+
+psa_status_t atecc608a_random(uint8_t *rand_out)
+{
+    psa_status_t status = PSA_ERROR_GENERIC_ERROR;
+
+    if (rand_out == NULL)
```

Comment:
Why bother checking a null pointer when you aren't even checking the buffer length?

Reply:
Fixed!